### PR TITLE
gitea smtp-gw section in the config

### PR DESCRIPTION
**Pull request by kucerakk**

### CODE REVIEW

**Review by gtema — DISMISSED** [review]

File: playbooks/roles/gitea/templates/app.ini.j2 [hunk 1]
```diff
@@ -345,9 +345,14 @@ ENABLE_AUTO_REGISTRATION = true
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;[mailer]
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+[mailer]
+ENABLED        = true
```
Comment (gtema) [inline playbooks/roles/gitea/templates/app.ini.j2:349]: maybe we should wrap this into at least "{% if gitea_smtp_host is defined %}"? And then maybe make type and tls configurable as well (with defaults)?

**Review by gtema — DISMISSED** [review]

File: playbooks/roles/gitea/templates/app.ini.j2 [hunk 2]
```diff
@@ -345,14 +345,16 @@ ENABLE_AUTO_REGISTRATION = true
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+{%if gitea_smtp_enable %}
 [mailer]
 ENABLED        = true
 FROM           = {{ gitea_smtp_from_address }}
```
Comment (gtema) [inline playbooks/roles/gitea/templates/app.ini.j2:351]: this perhaps should be renamed to gitea_mailer_from_address

File: playbooks/roles/gitea/templates/app.ini.j2 [hunk 3]
```diff
@@ -345,14 +345,16 @@ ENABLE_AUTO_REGISTRATION = true
...
-MAILER_TYPE    = smtp
+MAILER_TYPE    = {{ gitea_smtp_mailer_type }}
 HOST           = {{ gitea_smtp_host }}
```
Comment (gtema) [inline playbooks/roles/gitea/templates/app.ini.j2:353]: gitea_mailer_from

File: playbooks/roles/gitea/templates/app.ini.j2 [hunk 4]
```diff
@@ -345,14 +345,16 @@ ENABLE_AUTO_REGISTRATION = true
...
-IS_TLS_ENABLED = false
+IS_TLS_ENABLED = {{ gitea_smtp_tls }}
```
Comment (gtema) [inline playbooks/roles/gitea/templates/app.ini.j2:354]: gitea_mailer_is_tls_enabled

File: playbooks/roles/gitea/templates/app.ini.j2 [hunk 5]
```diff
@@ -345,14 +345,16 @@ ENABLE_AUTO_REGISTRATION = true
...
 USER           = {{ gitea_smtp_username }}
```
Comment (gtema) [inline playbooks/roles/gitea/templates/app.ini.j2:355]: gitea_mailer_user

File: playbooks/roles/gitea/templates/app.ini.j2 [hunk 6]
```diff
@@ -345,14 +345,16 @@ ENABLE_AUTO_REGISTRATION = true
...
 PASSWD         = {{ gitea_smtp_password }}
```
Comment (gtema) [inline playbooks/roles/gitea/templates/app.ini.j2:356]: gitea_mailer_passwd

File: playbooks/zuul/templates/group_vars/gitea.yaml.j2 [hunk 7]
```diff
@@ -3,7 +3,3 @@ gitea_secret_key: "dummy-secret-key"
 gitea_db_type: "sqlite3"
 gitea_cert: "gitea"
 
-gitea_smtp_from_address: "gitea@smtp.local"
```
Comment (gtema) [inline playbooks/zuul/templates/group_vars/gitea.yaml.j2]: and here they can be also becoming gitea_mailer_xxx

**Review by gtema — COMMENTED** [review]
approved, independent on the idea to make it more matching possibilities of gitea

**Review by gtema — DISMISSED** [review]

File: playbooks/zuul/templates/group_vars/gitea.yaml.j2 (hunk 7 above)
Comment (kucerakk) [reply playbooks/zuul/templates/group_vars/gitea.yaml.j2]: removed as in check job mailer part is disabled by default

**Review by gtema — APPROVED** [review]

**Pull request merged**